### PR TITLE
Cleanup includes and library path

### PR DESCRIPTION
**Pull request by MarkRiddoch**
Add additional library to package file

Signed-off-by: Mark Riddoch <mark@dianomic.com>

### CODE REVIEW

File: Package [hunk 1]
```diff
@@ -3,7 +3,7 @@
 plugin_name=iec104
 plugin_type=south
 plugin_install_dirname=iec104
-
+iadditional_libs="usr/local/lib:/usr/local/lib/liblib60870*"
```
Comment (YashTatkondawar) [inline Package:6]: `iadditional_libs `should be `additional_libs`

**Review by YashTatkondawar — CHANGES_REQUESTED** [review]
With this changes also no discovery in fledge. It is mostly because of wrong spelling of `additional_libs`

File: Package (hunk 1 above)
Comment (MarkRiddoch) [reply Package]: Thanks for picking that up

**Comment by YashTatkondawar** [comment]
Discovery is good now. One thing I noted is that whenever we create an iec104 south service it is printing **104 Server IP** in syslogs as warning.
**Syslogs:**
`Dec 4 05:06:26 ip-10-0-0-53 Fledge s1[10726]: WARNING: 127.0.0.1`

**Comment by MarkRiddoch** [comment]
> Discovery is good now. One thing I noted is that whenever we create an iec104 south service it is printing **104 Server IP** in syslogs as warning.
> **Syslogs:**
> `Dec 4 05:06:26 ip-10-0-0-53 Fledge s1[10726]: WARNING: 127.0.0.1`

That should probably be cleaned up, this is a third party generated plugin. At some point we will do some cleanup work on it and will look at this then. Thanks.

**Review by YashTatkondawar — CHANGES_REQUESTED** [review]
With latest commit discovery is again an issue with same error as previous:
`Dec 10 06:21:11 ip-10-0-0-53 Fledge[15562] ERROR: utils: fledge.services.core.api.utils: Expecting value: line 1 column 1 (char 0)`

**Comment by MarkRiddoch** [comment]
> With latest commit discovery is again an issue with same error as previous:
> `Dec 10 06:21:11 ip-10-0-0-53 Fledge[15562] ERROR: utils: fledge.services.core.api.utils: Expecting value: line 1 column 1 (char 0)`

It doesn't do this for me. Are you sure you have the right version? Can you attach an ldd output for the plugin please?

ldd ${FLEDGE_ROOT}/plugins/south/iec104/libiec104.so

**Comment by YashTatkondawar** [comment]
> > With latest commit discovery is again an issue with same error as previous:
> > `Dec 10 06:21:11 ip-10-0-0-53 Fledge[15562] ERROR: utils: fledge.services.core.api.utils: Expecting value: line 1 column 1 (char 0)`
> 
> It doesn't do this for me. Are you sure you have the right version? Can you attach an ldd output for the plugin please?
> 
> ldd ${FLEDGE_ROOT}/plugins/south/iec104/libiec104.so

Now with this branch also it is giving same compilation errors that I have reported in FOGL-4943.

**Comment by MarkRiddoch** [comment]
A li esce issue has come to light that prevents us creating packages for this plugin. The package generation for this plugin is being removed.

**Review by YashTatkondawar — APPROVED** [review]
Currently, the packaging of this plugin was failing. But as of now we will not be packaging this plugin so I will exclude it from all packaging jobs.

**Pull request merged**